nevergrad/benchmark/experiments.py

```diff
@@ -1139,6 +1140,21 @@ def images(seed: tp.Optional[int] = None) -> tp.Iterator[Experiment]:
                         yield xp
 
 
+@registry.register
+def imagesUsingGan(seed: tp.Optional[int] = None) -> tp.Iterator[Experiment]:
+    """Optimizing an image using koncept512 and a GAN"""
+    seedg = create_seed_generator(seed)
+    optims = ["CMA", "Shiwa", "DE", "PSO", "RecES", "RecMixES", "RecMutDE", "NSGAIIES", "ParametrizationDE"]
+    if default_optims is not None:
+        optims = default_optims
+    for budget in [100 * 5 ** k for k in range(3)]:
+        for num_workers in [1]:
+            for algo in optims:
+                for func in [ImageFromPGAN()]:
```

Comment:
please create the func outside all the for loops, and no need to iterate if there is only one of them (same for num_workers)